internal/cmd/flags.go

```diff
@@ -45,17 +45,18 @@ var (
 		Usage:   "original reference",
 	}
 
-	FlagPluginConfig = &cli.StringSliceFlag{
+	FlagPluginConfig = &cli.StringFlag{
 		Name:    "pluginConfig",
 		Aliases: []string{"pc"},
 		Usage:   "list of comma-separated {key}={value} pairs that are passed as is to the plugin, refer plugin documentation to set appropriate values",
 	}
 )
 
-func ParseFlagPluginConfig(pluginConfigSlice []string) (map[string]string, error) {
-	if len(pluginConfigSlice) == 0 {
+func ParseFlagPluginConfig(v string) (map[string]string, error) {
+	if v == "" {
 		return nil, nil
 	}
+	pluginConfigSlice := strings.Split(v, ",")
```

Comment:
If we directly split the string we won't handle cases where the plugin param value itself contains a delimiter, correct?

`--pluginConfig a=b, c="value1,value2"`

Reply:
Good shoot. Fixed in https://github.com/notaryproject/notation/pull/181/commits/e173eb55240d045caa62acdfa0c57f3befc002c6.